LuaUI/Widgets/gui_selectedunits_gl4.lua

```diff
@@ -7,7 +7,7 @@ function widget:GetInfo()
 		license = "GNU GPL, v2 or later",
 		-- Somewhere between layer -40 and -30 GetUnitUnderCursor starts
 		-- returning nil before GetUnitsInSelectionBox includes that unit.
-		layer = -30,
+		layer = -8, -- Draw below health bars and selection.
```

Comment:
selection is this one
```suggestion
		layer = -8, -- Draw below health bars, icons, and nametags
```

Reply:
Oops. Is it a good idea to enumerate all the options (which isn't a complete list here anyway). Or I suppose we could import constants?